GameBoard/HostedServices/ScopedHostedService.cs

```diff
@@ -7,16 +7,16 @@ namespace GameBoard.HostedServices
 {
     public abstract class ScopedHostedService : HostedService
     {
-        private readonly IServiceScopeFactory _serviceScopeFactory;
+        protected readonly IServiceScopeFactory ServiceScopeFactory;
```

Comment:
Why does it start with an underline and a capital letter?

Reply:
`protected` fields should be called like that.

To be honest, `protected` fields shouldn't even exist, as they break encapsulation and one should use properties in their place.

But after the change to `CronScheduledService` this remains `private readonly`.